pom.xml

```diff
@@ -173,7 +118,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-release-plugin</artifactId>
-                    <version>2.5.3</version>
+                    <version>${version.plugin.release}</version>
```

Comment:
I think this plugin can be removed.

Reply:
The plugin `org.eclipse.cbi.maven.plugins` can be removed as well, I believe.

Reply:
@Emily-Jiang Need to also remove `microprofile-maven-build-extension` coming from mentioned custom `<pluginRepositories>`, are we good without that plexus component [descriptor](https://github.com/eclipse/microprofile/blob/master/attic/build-tools/maven-build-extension/src/main/resources/META-INF/plexus/components.xml)?

Reply:
hmmm. I think the descriptor in the attic is going to be removed. Can you build the pdf to see whether all images are rendered ok.

Reply:
Looks ok 
![image](https://user-images.githubusercontent.com/1773630/148944666-f9f6d7dc-0005-456e-acb0-e91195b7b94f.png)

Reply:
regenerated marble diagrams looks ok too